packages/backend/src/ee/models/AiAgentModel.ts

```diff
@@ -894,6 +912,14 @@ export class AiAgentModel {
                     filtersOutput: row.filters_output,
                     metricQuery: row.metric_query,
                     humanScore: row.human_score,
+                    artifact: row.ai_artifact_uuid
+                        ? {
+                              uuid: row.ai_artifact_uuid,
+                              versionNumber: row.version_number!,
```

Comment:
can we use safe defaults just in case? 

```suggestion
                              versionNumber: row.version_number ?? 1,
```